docs/index.md

```diff
@@ -0,0 +1,70 @@
+# Iterative Provider
+
+Use the Iterative Provider to launch resource-intensive tasks in popular cloud
+providers with just a simple Terraform file.
```

Comment:
p.s. I noticed docs/guides/getting-started.md is not wrapped at 80 chars. Seems inconsistent 💅

Reply:
BTW, minor but this can be read as condescending.
```suggestion
Use the Iterative Provider to launch resource-intensive tasks in popular cloud
providers with a single Terraform file.
```